docs/reference/contrib/modeladmin/indexview.rst

```diff
@@ -631,7 +631,7 @@ change a few attributes to change the thumbnail to your liking, like so:
 
     class Person(models.Model):
         name = models.CharField(max_length=255)
-        avatar = models.ForeignKey('wagtailimages.Image', on_delete=models.SET_NULL, null=True)
+        avatar = models.ForeignKey('media.Image', on_delete=models.SET_NULL, null=True)
```

Comment:
It might be better to encourage use of `wagtail.images.get_image_model_string()` here instead